src/runtimes/docker/contracts/ContainerClient.ts

```diff
@@ -744,6 +744,10 @@ export type RunContainerCommandOptions = CommonCommandOptions & {
      * Optional command to use in starting the container
      */
     command?: Array<string> | string;
+    /**
+     * Optional expose ports for the container
+     */
+    exposes?: Array<number>;
```

Comment:
I don't feel strongly one way or the other, but maybe "expose"? Most of the other array arguments have a plural, but "exposes" isn't a plural...I don't know that there is a plural of "expose".

Reply:
Maybe exposePorts?

Reply:
good point! and I think`exposePorts` make the most sense to me